appium/tests/specs/mock/setup/SetupOnlyRevokedKeyFromEKM.spec.ts

```diff
@@ -21,10 +21,10 @@ describe('SETUP: ', () => {
     const emailSubject = CommonData.simpleEmail.subject;
     const emailText = CommonData.simpleEmail.message;
 
+    // When private key is revoked key, there are no public keys to pick. So missing sender public key error occurs.
     const noPrivateKeyError = 'Error\n' +
-      'Could not compose message\n' +
-      '\n' +
-      'Error: Error encrypting message: Could not find valid key packet for signing in key bf79556b2cad5c1e';
+      'Could not compose message\n\n' +
+      'Missing sender public key. Is FlowCrypt iOS app well set up?';
```

Comment:
This is not a very good error.

There are actually two separate concerns when sending a signed+encrypted message, as it relates to my own keys:
 - do I have any **private** key that is usable for **signing**? -> error `Your account keys are not usable for signing`
 - do I have any **public** key that is usable for **encryption**? -> error `Your account keys are not usable for encryption`

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/1546#issuecomment-1111980975

Reply:
At least for now, `Missing sender public key. Is FlowCrypt iOS app well set up?` should become `None of your account public keys are usable for encryption.` which will be slightly more correct than acting as if there are no keys (eg if one of them is revoked or expired).

Reply:
yeah, implemented that way.